### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by estherkim**
This will require Yuliia to approve stable and 1% promote PRs as part of the QA process

cc @JuliaTrubchyk

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# QA approval for 1% and stable promotions
+configs/versions.json    @JuliaTrubchyk
```
Comment (estherkim) [inline CODEOWNERS:2]: @rsimha thoughts on creating a QA team instead?

**Review by danielrozenberg — APPROVED** [review]

**Comment by danielrozenberg** [comment]
Expand the comment to mention that this is branch-protected, not for every change to versions.json

File: CODEOWNERS (hunk 1 above)
Comment (rsimha) [reply CODEOWNERS:4]: Good idea. I just created https://github.com/orgs/ampproject/teams/amp-qa and invited @JuliaTrubchyk to the team. Let me know if this works.

Comment (estherkim) [reply CODEOWNERS]: It should actually just be QA. Thanks!

Comment (rsimha) [reply CODEOWNERS]: Gotcha. Renamed to https://github.com/orgs/ampproject/teams/qa.

Comment (estherkim) [reply CODEOWNERS]: Wait haha I meant the members of the team, not the name. amp-qa is fine!

Comment (rsimha) [reply CODEOWNERS]: 🤦 Renamed back to `amp-qa`.

**Pull request merged**